### DISCUSSION TITLE
Setup a minimun zoom in config file

### DISCUSSION THREAD

**Discussion by phpia** [maplibre/martin#252] [Ideas]
It would be nice if it will be possible to limit the minimum zoom at wich martin return tiles.

I know that Im able to do it from javascript in leaflet or mapbox but anyone is able to bypass this control and could perform like dddos attak against postgres 

Thanks!

**Comment by douglasg14b** [comment]
This would be very good to have. Though a workaround for now could be to prevent such requests from your reverse proxy? NGINX for example can use regex patterns.

**Comment by phpia** [comment]
This is the documentation in geoserver for this feature. Geoserver uses the concept of scale. 
Im not sure if the scale is evaluated before send query to postgres or if it is a condition in the query.

https://docs.geoserver.org/latest/en/user/styling/sld/reference/rules.html#scale-selection
https://docs.geoserver.org/latest/en/user/styling/ysld/reference/scalezoom.html

**Comment by stepankuzmin** [comment]
Hi everyone,

I'm working on it in #265

**Comment by phpia** [comment]
Hi @stepankuzmin I have tested the feature in alpha version 1.0

It works nice!

Thanks a lot, nice work

**Reply by stepankuzmin** [reply]
Great, thanks!
